From 4dbdd4778d97e0b8215754d14c209c6c430e473d Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Sat, 17 Sep 2016 21:36:45 +0200
Subject: [PATCH] Install binaries, libraries, headers files into framework

---
 CMakeLists.txt | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 19eba46..cdaff60 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,6 +84,7 @@ if(APPLE)
   if(BUILD_FRAMEWORK)
       set(BUILD_LIBPYTHON_SHARED ON CACHE BOOL "Forced to ON because of BUILD_FRAMEWORK=ON" FORCE)
   endif()
+  set(FRAMEWORK_DIR_PREFIX Python.framework/Versions/${PY_VERSION_MAJOR}.${PY_VERSION_MINOR})
 else()
   set(BUILD_FRAMEWORK OFF)
 endif()
@@ -318,6 +319,9 @@ if(USE_LIB64)
     set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS ON)
     set(LIBDIR "lib64")
 endif()
+if(BUILD_FRAMEWORK)
+    set(LIBDIR ${FRAMEWORK_DIR_PREFIX}/lib)
+endif()
 set(PYTHONHOME "${LIBDIR}/${LIBPYTHON}")
 
 # Install tree directory
@@ -325,15 +329,22 @@ set(BIN_INSTALL_DIR bin)                         # Contains the python executabl
 if(INSTALL_WINDOWS_TRADITIONAL)
     set(BIN_INSTALL_DIR .)                       # Contains the python executable
 endif()
+if(BUILD_FRAMEWORK)
+    set(BIN_INSTALL_DIR ${FRAMEWORK_DIR_PREFIX}/bin)
+endif()
 set(CONFIG_INSTALL_DIR share/${LIBPYTHON})
 set(EXTENSION_INSTALL_DIR ${PYTHONHOME}/lib-dynload)
 if(INSTALL_WINDOWS_TRADITIONAL)
     set(EXTENSION_INSTALL_DIR DLLs)
 endif()
+
 set(INCLUDE_INSTALL_DIR include/${LIBPYTHON})
 if(MSVC)
     set(INCLUDE_INSTALL_DIR include/python${PY_VERSION_MAJOR}.${PY_VERSION_MINOR})
 endif()
+if(BUILD_FRAMEWORK)
+    set(INCLUDE_INSTALL_DIR ${FRAMEWORK_DIR_PREFIX}/include/python${PY_VERSION_MAJOR}.${PY_VERSION_MINOR})
+endif()
 # Build tree directory
 set(BIN_BUILD_DIR ${PROJECT_BINARY_DIR}/bin)
 set(CONFIG_BUILD_DIR ${PROJECT_BINARY_DIR}/${CONFIG_INSTALL_DIR})
-- 
2.7.4

